nordicsemi/dfu/package.py

```diff
@@ -187,6 +185,20 @@ def __init__(self,
                                      boot_validation_type=app_boot_validation_type,
                                      init_packet_data=init_packet_vars)
 
+        # WARNING
+        # The code 0x00 in the sd_req field translates to deleting the
+        # softdevice. Moving the setting of the `REQUIRED_SOFTDEVICES_ARRAY`
+        # field to be `sd_req` to before the `self__.add_firmware_info` call
+        # for HexType.EXTERNAL_APPLICATION can lead customers potentially
+        # bricking their devices.  There may be configurations that avoid the
```

Comment:
I'd explicitly say "bricking their devices when trying to update the app software", to emphasize that it's ok to use a 0 `sd_req` for bootloader and softdevice changes (assuming it actually is, of course).